about.html

```diff
@@ -13,52 +13,88 @@
 
 <body>
     <div class="container about-page">
-     
-    About
-
-    Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.
 
-
-    <img src="img/about-plan.png" alt="strategy">
+        <header class="head-cont-about">
+            <Img class="logo-top-about" src="./img/lambda-black.png"></Img>
     
-    Strategy
+            <nav class="nav">
+                <a href="index.html">Home</a>
+                <a href="#">About</a>
+                <a href="#">Products</a>
+                <a href="#">Blog</a>
+                <a href="#">Contact</a>
+            </nav>
+        </header>
     
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
-
-    Learn More
-
-
+        <div class="jumbo-div-about">
+        <img class="jumbo-img-about" src="./img/jumbo-about.png"></img>
+        </div>
+     
+        <section class="Top-about">
+    <h2>About</h2>
+   <p>Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.</p>
+  </section>
 
+  <Section class="about-logo-mid">
+    <img src="img/about-plan.png" alt="strategy">
     <img src="img/about-working.png" alt="strategy">
+  </section>
     
```

Comment:
in these next two areas you have one section tag opening capitalized and closing lowercase (<Section></section>) then below the exact opposite (<section> </Section>) be sure in regular html to keep the tags lowercase. once we get into JS components in libraries like react, we will capitalize out components, but the native html tags should be lowercased